### ISSUE TITLE
Silvanus redone using...

### ISSUE THREAD

**Issue by olonu** [arianne/stendhal#114]
Silvanus redone using current elf wizard.  Changed some coloration and hair to hair from Townfolk-Old-M-001 light.png.  
-Still needs wand; tier towards earthly and preferably some greenish, earth energy around top
![rsilvanus](https://user-images.githubusercontent.com/37004874/38168664-601dc7f2-3521-11e8-8cf5-d1b77d048cb7.png)

**Comment by AntumDeluge** [comment]
I had [copied over the *elf_wizard* sprite](https://github.com/arianne/stendhal/commit/b49262d) for Silvanus. But, I like that you have redone him so he looks different. Thank you.

Committed to master branch: 79b054f

**Comment by AntumDeluge** [comment]
I guess since you said he still needs wand, I will open this back up.

**Comment by olonu** [comment]
I'm probably not getting one unless I find a good one.  I tried those rotating orbs for the energy effect, but then I read the description for the green one and it said "venom".  Also the rotating orbs don't scale well.  I wanted to post this because I had already went through deciding about Silvanus earlier before dress 099 had to be removed because I was going to use dress 099, but now I'll leave it in the mage outfit.  I consider it done for now unless someone else has a good earthly wand; it doesn't have to have energy effects.
